framework/src/main/java/org/checkerframework/framework/type/AbstractViewpointAdapter.java

```diff
@@ -47,6 +49,9 @@ public abstract class AbstractViewpointAdapter implements ViewpointAdapter {
     /** The annotated type factory. */
     protected final AnnotatedTypeFactory atypeFactory;
 
+    /** The set of type parameter that have been visited. */
```

Comment:
The `type parameter` is misleading... all visited types are added, not just type parameters. If it is only for type parameters, can you use a more specific type?